src/main/java/ru/skypro/homework/entity/User.java

```diff
@@ -0,0 +1,78 @@
+package ru.skypro.homework.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import javax.management.relation.Role;
+import javax.persistence.*;
+import java.util.Collection;
+
+/**
+ * Сущность пользователя
+ */
+@Entity
+@Data
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name="users")
+public class User {
+    /**
+     * Id пользователя
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name="id")
+    private Integer id;
+
+    /**
+     * Логин пользователя
+     */
+    @Column(name = "user_name")
+    private String userName;
+    /**
+     * mail пользователя
+     */
+    @Column(name="email",  nullable = false, unique = true)
+    private String email;
+    /**
+     * имя пользователя
+     */
+    @Column(name="first_name")
+    private String firstName;
+    /**
+     * фамилия пользователя
+     */
+    @Column(name = "last_name")
+    private String lastName;
+
+    /**
+     * телефон пользователя
+     */
+    @Column(name="phone", length = 20)
+    private String phone;
+    /**
+     * права доступа пользователя
+     */
+    @Column(name="user_role", nullable = false)
+    //@Enumerated(EnumType.STRING)
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
```
нужно раскомментировать